kokoro/ubuntu/release.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+# create a timestamp file
+date +%s > timestamp.txt
+
+cd git/kokoro-codelab-elharo
```

Comment:
It looks odd to embed a particular user id in a script instead of `$USER`.  But that could be because I know nothing about kokoro!